ZohoCRMCharts/Classes/ZCRMComparator.swift

```diff
@@ -0,0 +1,637 @@
+		self.scrollView.bounces = false
+		var constraints: [NSLayoutConstraint] = []
+		
+		constraints += NSLayoutConstraint.constraints(withVisualFormat: "H:|[chunksView(==chunksWidth)][scrollView]|", options: [], metrics: ["chunksWidth": self.chunksWidth], views: ["chunksView": self.chunksView, "scrollView": self.scrollView])
+		constraints += NSLayoutConstraint.constraints(withVisualFormat: "V:[title]-[scrollView]-0-|", options: [], metrics: nil, views: ["title": self.titleView, "scrollView": self.scrollView])
+		constraints += NSLayoutConstraint.constraints(withVisualFormat: "V:[title]-[chunksLabel]-0-|", options: [], metrics: nil, views: ["title": self.titleView, "chunksLabel": self.chunksView])
+		constraints += NSLayoutConstraint.constraints(withVisualFormat: "H:|-0-[headers(==\(scollViewCSWidth))]-0-|", options: [], metrics: nil, views: ["headers": self.headersView])
+		constraints += NSLayoutConstraint.constraints(withVisualFormat: "H:|-0-[collectionView(==\(scollViewCSWidth))]-0-|", options: [], metrics: nil, views: ["collectionView": self.collectionView])
+		self.activate(constraints: constraints)
+		self.chunksView.layoutIfNeeded()
+		constraints.removeAll()
+		
+		let headerHeight: CGFloat = self.chunksView.frame.height / (self.rowCount + 1).toCGFloat()
+		constraints += NSLayoutConstraint.constraints(withVisualFormat: "V:|-0-[headers(headerHeight)]-0-[collectionView(cVHeight)]-0-|", options: [], metrics: ["headerHeight": headerHeight, "cVHeight": self.chunksView.frame.height - headerHeight], views: ["headers": self.headersView, "collectionView": self.collectionView])
+		self.activate(constraints: constraints, true)
+		self.setSportComparatorConfigs()
+	}
+	
+	private func setSportComparatorConfigs() {
+		
+		let headerHeight: CGFloat = self.chunksView.frame.height / (self.rowCount + 1).toCGFloat()
+		let cVHeight: CGFloat = self.chunksView.frame.height - headerHeight
+		let horizontalSpacing = self.cellWidth * 0.2
+		let verticalSpacing = (cVHeight * 0.2) / self.columCount.toCGFloat()
+		let cellHeight = cVHeight * 0.8 / self.rowCount.toCGFloat()
+		
+		self.scrollView.contentSize.height = cVHeight
+		self.headersView.layoutMargins =  UIEdgeInsets(top: 0, left: horizontalSpacing , bottom: 0, right: horizontalSpacing)
+		self.headersView.spacing = horizontalSpacing
+		self.headersView.isLayoutMarginsRelativeArrangement = true
+		self.collectionViewFlowLayout.itemSize = CGSize(width: self.cellWidth, height: cellHeight)
+		self.collectionViewFlowLayout.sectionInset = UIEdgeInsets(top: verticalSpacing, left: horizontalSpacing, bottom: verticalSpacing, right: horizontalSpacing)
+	}
+	
+	private func addElegantComparatorConstraints() {
+		
+		let scollViewCSWidth = self.cellWidth * self.columCount.toCGFloat()
+		self.scrollView.contentSize.width = scollViewCSWidth
+		self.chunksView.layoutMargins = UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 10)
```

Comment:
why margin insets are not equal?

Reply:
No need for that margin. It has been removed